templates/repo/issue/card.tmpl

```diff
@@ -49,6 +49,13 @@
 		</div>
 		{{end}}
 		{{end}}
+		{{$tasks := .GetTasks}}
+		{{if gt $tasks 0}}
+			<div class="meta gt-my-2">
+				{{svg "octicon-checklist" 16 "gt-mr-2 gt-vm"}}
+				<span class="gt-vm">{{.GetTasksDone}} / {{$tasks}}</span>
+			</div>
+		{{end}}
```

Comment:
Can we perhaps extract a subtemplate instead of manually adding it each time?

Reply:
"each time"?

Reply:
issue/PR list, milestones, projects, …

Reply:
They already share a template.

Reply:
Sure, but the classes are totally different.

Reply:
Ideally make classes the same, but we can do later I guess then.